xunit-autolabeler-v2/ast_parser/python/README.md

```diff
@@ -0,0 +1,15 @@
+# AST snippet parser - Python
+
+## What this does
+This tool extracts snippet data from Python snippets [such as these](https://github.com/googlecloudplatform/python-docs-samples), and serializes that data into a language-agnostic JSON format.
+
+That data can then be used by the "meta-parser" in `ast_parser/core` to match test results with their associated snippets/region tags.
+
+## Usage
+Run the following command to generate a `repo.json` file for a specific directory:
+
+```
+python python_bootstrap.py YOUR_SAMPLE_DIR
```

Comment:
When I try to run `python_bootstrap.py`, I got this error:

```
$ python python_bootstrap.py sample
Traceback (most recent call last):
  File "python_bootstrap.py", line 19, in <module>
    from python import invoker
ModuleNotFoundError: No module named 'python'
```

If you want to add this file, please add a test too.

Reply:
Removed it instead. (I'll add it in a later PR, as discussed above.)